log/logger.go

```diff
@@ -1,25 +1,28 @@
 package log
 
 import (
-	"bytes"
 	"io"
 	"log"
 	"os"
 	"path/filepath"
 
 	pkgerrors "github.com/pkg/errors"
+	"github.com/zimmski/osutil/bytesutil"
 )
 
+// Logger holds a logger to log to.
+type Logger = log.Logger
+
 // Buffer returns a logger that writes to a buffer.
-func Buffer() (buffer *bytes.Buffer, logger *log.Logger) {
-	buffer = new(bytes.Buffer)
+func Buffer() (buffer *bytesutil.SynchronizedBuffer, logger *Logger) {
+	buffer = new(bytesutil.SynchronizedBuffer)
 	logger = log.New(buffer, "", log.LstdFlags)
 
 	return buffer, logger
 }
 
 // File returns a logger that writes to a file.
-func File(path string) (logger *log.Logger, loggerClose func(), err error) {
+func File(path string) (logger *Logger, loggerClose func(), err error) {
```

Comment:
Not sure if we use this function anywhere now?

Reply:
Keep it for now.